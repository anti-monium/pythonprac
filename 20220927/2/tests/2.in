9,7,4,2,3,1,5,8,6,111,256,128,344,343,48,36
