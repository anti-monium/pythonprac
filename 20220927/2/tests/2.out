[1, 2, 48, 3, 4, 111, 5, 256, 344, 6, 343, 7, 8, 9, 128, 36]
